src/context.rs

```diff
@@ -89,6 +92,7 @@ impl Context {
     pub fn new() -> Context {
         let mut ctx = Context {
             debug_mode: false,
+            code: String::new(), // FIXME: Is this the correct behavior?
```

Comment:
I think an `Option` would be a good choice here

Reply:
Done in 05e98bd